### ISSUE TITLE
Is this genuine copy in google play ?

### ISSUE THREAD

**Issue by izac3d** [GoogleChromeLabs/squoosh#1404]
https://play.google.com/store/apps/details?id=com.ailabhub.squoosh&hl=en&gl=US

**Comment by surma** [comment]
No, this is not us. As  you can tell by the screenshots it’s outdated. 

I’ll flag it with someone.

**Comment by william34561** [comment]
This is disgusting it is the second time I find someone want to take advantage of your great software . Is there a way to prevent turning this app into android app from js code ?

Also I found this
https://squoosh-desktop.vercel.app

**Comment by william34561** [comment]
I suggest to end availability as open source becuase it seems this hijack will not end

**Comment by SanujaNS** [comment]
> I suggest to end availability as open source becuase it seems this hijack will not end

Intriguing how the urge to air grievances about open source can lead one to cross paths with GitHub, creating an account just for this purpose 😞

**Comment by izac3d** [comment]
> > I suggest to end availability as open source becuase it seems this hijack will not end
> 
> Intriguing how the urge to air grievances about open source can lead one to cross paths with GitHub, creating an account just for this purpose 😞

No this is just your imagination

**Comment by thibaultmol** [comment]
@surma another one btw https://play.google.com/store/apps/details?id=com.toolyoulove.compressimage
